parquet/read.go

```diff
@@ -80,7 +78,22 @@ func reverseTransformArray(dt arrow.DataType, arr arrow.Array) arrow.Array {
 	case *array.Time64:
 		return reverseTransformTime64(dt.(*arrow.Time64Type), arr)
 	case *array.Struct:
-		return reverseTransformStruct(dt.(*arrow.StructType), arr)
+		dt := dt.(*arrow.StructType)
+		children := make([]arrow.ArrayData, arr.NumField())
+		names := make([]string, arr.NumField())
+		for i := range children {
+			// struct fields can be odd when read from parquet, but the data is intact
+			child := array.MakeFromData(arr.Data().Children()[i])
+			children[i] = reverseTransformArray(dt.Field(i).Type, child).Data()
+			names[i] = dt.Field(i).Name
```

Comment:
`names` isn't used

Reply:
Thanks!
Removed in bc6ccee6a427ab5729bc01413f34057293f08add